### PR TITLE
Fix severity check box not filter items correctly & severity not counted properly 

### LINKED ISSUE #11458: [UI] Notification center not being filtered out based on Event type.
### Current Behavior

In Notification Center the selectors for Info, Error, Success, Warnings don't filter out the events based on the Event Type.
### Desired Behavior

It should filter the events based on the Event type.
### Screenshots/Logs

Screenshot from 2024-07-25 05-24-45

### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by jerensl**
**Notes for Reviewers**

This PR fixes #11458 

Screenshot_26-7-2024_193715_localhost
Screenshot_26-7-2024_193656_localhost

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by jerensl** [comment]
I also add fix for the severity checkbox not counted, which there a problem in the database

**Comment by leecalcote** [comment]
Yay, @jerensl! 🎈

File: server/models/events_persister.go [hunk 1]
```diff
@@ -154,13 +154,13 @@ func (e *EventsPersister) PersistEvent(event *events.Event) error {
...
 
-func (e *EventsPersister) getCountBySeverity(userID uuid.UUID, eventStatus events.EventStatus) ([]*CountBySeverityLevel, error) {
+func (e *EventsPersister) getCountBySeverity(userID, sysID uuid.UUID, eventStatus events.EventStatus) ([]*CountBySeverityLevel, error) {
 	if eventStatus == "" {
 		eventStatus = events.Unread
 	}
 
 	eventsBySeverity := []*CountBySeverityLevel{}
-	err := e.DB.Model(&events.Event{}).Select("severity, count(severity) as count").Where("status = ? and user_id = ?", eventStatus, userID).Group("severity").Find(&eventsBySeverity).Error
+	err := e.DB.Model(&events.Event{}).Select("severity, count(severity) as count").Where("status = ? AND (user_id = ? OR user_id = ?)", eventStatus, userID, sysID).Group("severity").Find(&eventsBySeverity).Error
```
Comment (codeSafari10) [inline server/models/events_persister.go:163]: @jerensl  why is or agains user_id and user_id . i think it should be system_id .

Comment (aabidsofi19) [inline server/models/events_persister.go:163]: @jerensl why is or against user_id and user_id . i think it should be system_id .

**Review by aabidsofi19 — CHANGES_REQUESTED** [review]

File: server/models/events_persister.go (hunk 1 above)
Comment (jerensl) [reply server/models/events_persister.go]: @aabidsofi19 I just wonder if maybe there is a bug I didn't know, that's why I followed the same approach as the previous PR https://github.com/meshery/meshery/pull/11065/files#diff-94ae26e2758ca9153f353b080834d204c2a8a920a98d271b66cf251efb54b13e

Comment (jerensl) [reply server/models/events_persister.go]: I also made the change to the system_id in the latest commit and it works in my local as well

**Comment by vishalvivekm** [comment]
@jerensl 
Let's discuss this on the Meshery dev meeting starting 9 AM Central time today. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk)

**Comment by jerensl** [comment]
> @jerensl Let's discuss this on the Meshery dev meeting starting 9 AM Central time today. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk)

Hi @vishalvivekm I think I'm missing out on explaining my PR at the last meeting and also attending the meeting, can I do it here or do I need to wait for the next meeting?

**Comment by Ashparshp** [comment]
@jerensl, let's discuss this PR during the Meshery dev meeting call, if you would. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk)

**Pull request closed without merging**